src/wallet/wallet2.h

```diff
@@ -365,6 +422,41 @@ namespace tools
       const cryptonote::account_public_address &account_public_address,
       const crypto::secret_key& viewkey = crypto::secret_key());
     /*!
+     * \brief Creates a multisig wallet
+     * \return empty if done, non empty if we need to send another string
+     * to other participants
+     */
+    std::string make_multisig(const std::string &password,
+      const std::vector<crypto::secret_key> &view_keys,
+      const std::vector<crypto::public_key> &spend_keys,
+      uint32_t threshold);
+    /*!
+     * \brief Finalizes creation of a multisig wallet
+     */
+    bool finalize_multisig(const std::string &password,const std::unordered_set<crypto::public_key> &pkeys, const std::vector<crypto::public_key> &signers);
+    /*!
+     * Get a packaged multisig information string
+     */
+    std::string get_multisig_info() const;
+    /*!
+     * Verifies and extracts keys from a packaged multisig information string
+     */
+    bool verify_multisig_info(const std::string &data, crypto::secret_key &skey, crypto::public_key &pkey) const;
```

Comment:
Neither this function nor the other `_extra_` function below seem dependent on the `wallet2` member data, so shouldn't they be made `static` functions, or be moved into `multisig.cpp`?

Reply:
I see multisig.cpp as crypto helpers or building blocks. The two functions here deal with a particular encoded piece of data, which I don't think belongs there. I guess the functions could be static if they don't use object data. I don't have much preference so I'll change it so.